include/envoy/runtime/runtime.h

```diff
@@ -102,6 +102,19 @@ class Snapshot {
   virtual bool featureEnabled(const std::string& key, uint64_t default_value,
                               uint64_t random_value) const PURE;
 
+  /**
+   * Test if a feature is enabled using the built in random generator and total number of buckets
+   * for sampling.
+   * @param key supplies the feature key to lookup.
+   * @param default_value supplies the default value that will be used if either the feature key
+   *        does not exist or it is not an integer.
+   * @param num_buckets control max number of buckets for sampling. Sampled value will be in a range
+   * of [0, num_buckets).
+   * @return true if the feature is enabled.
+   */
+  virtual bool featureEnabledEx(const std::string& key, uint64_t default_value,
+                                uint64_t num_buckets) const PURE;
```

Comment:
Is there any callsite that is calling the 3 arg form of featureEnabledEx supplying buckets instead of random value? If not, lets get rid of this function.

Reply:
+1 if possible.

Reply:
Addressed via aef3da0 and c31bd82.